packages/color-scale-block/src/ColorScaleBlock.tsx

```diff
@@ -431,6 +431,13 @@ export const ColorScaleBlock: FC<BlockProps> = ({ appBridge }) => {
                 ref={colorScaleBlockRef}
                 data-test-id="color-scale-block"
                 className="tw-w-full tw-p-px tw-mb-4 tw-border tw-border-line tw-rounded"
+                style={{
+                    height: `${parseInt(colorScaleHeight) + COLOR_SCALE_BLOCK_BORDER_WIDTH * 2}px`,
```

Comment:
looks a bit hacky :D not sure, but could you just use `height: '100%'`? (tw-h-full)

Reply:
unfortunately I can't do 100% height because the height is able to change via the settings